main/rpc/index.js

```diff
@@ -169,7 +169,7 @@ const rpc = {
   },
   createFromAddress (address, cb) {
     if (!utils.isAddress(address)) return cb(new Error('Invalid Address'))
-    accounts.add(address, { type: 'Address' })
+    accounts.add(address, '', { type: 'Address' })
```

Comment:
I think we should pass the name down from whatever calls this to give the account a proper name, similar to the function below